lizmap/modules/lizmap/lib/Request/WMTSRequest.php

```diff
@@ -98,7 +98,7 @@ public function gettile()
 
         foreach ($params as $var => $param) {
             ${$var} = $this->param($param);
-            if (!${$var}) {
+            if (!${$var} && !is_numeric(${$var})) {
```

Comment:
As LayerName is not numeric (I guess), so it will fail because of LayerName.

I suggest to have this test instead: `if (${$var} === '' || ${$var} === null) {`

Reply:
Thanks changes done. I will rebase the code.